### PR TITLE
Add Terms Page

### PR DESCRIPTION
**Pull request by CarlGraff**
Let me know if we should add an open source license here. Maybe it's only needed for the source code in the repo and not in browser.

Issue #44

### CODE REVIEW

**Comment by NickEngmann** [comment]
can you throw up a picture of how the terms page looks like?

**Comment by NickEngmann** [comment]
Screen Shot 2019-06-12 at 6 27 08 PM

File: p2plending/frontend/src/app/terms/Terms.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+              automatically apply to you - you should make sure therefore
+              that you read them carefully before using the app. 
+            </p> 
+            <p>
+              Open San Diego Org is committed to ensuring that the app
+              is as useful and efficient as possible. For that reason, we
+              reserve the right to make changes to the app at any time and for any reason.
+            </p> 
+            <p>
+              The Lendo app stores and processes personal data
```
Comment (NickEngmann) [inline p2plending/frontend/src/app/terms/Terms.js]: Replace "The Lendo app" with { title }

File: p2plending/frontend/src/app/terms/Terms.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        <div className="container">
+          <div className="p-4 rounded mx-auto" >
+            <h2>Terms</h2>
+             <p>
+              By downloading or using the app, these terms will
+              automatically apply to you - you should make sure therefore
+              that you read them carefully before using the app. 
+            </p> 
+            <p>
+              Open San Diego Org is committed to ensuring that the app
```
Comment (NickEngmann) [inline p2plending/frontend/src/app/terms/Terms.js]: Replace "Open San Diego Org" with "Open San Diego"

File: p2plending/frontend/src/app/terms/Terms.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+              process of removing software restrictions and limitations
+              imposed by the official operating system of your device. It
+              could make your phone vulnerable to
+              malware/viruses/malicious programs, compromise your phone's
+              security features and it could mean that the
+              Lendo app won't work properly or at all.
+            </p> 
+            <p>
+              You should be aware that there are certain things that
+              Open San Diego Org will not take responsibility for.
```
Comment (NickEngmann) [inline p2plending/frontend/src/app/terms/Terms.js]: Replace "Open San Diego Org" with "Open San Diego"

File: p2plending/frontend/src/app/terms/Terms.js [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+              accepting responsibility for any such charges, including
+              roaming data charges if you use the app outside of your home
+              territory (i.e. region or country) without turning off data
+              roaming. If you are not the bill payer for the device on
+              which you're using the app, please be aware that we assume
+              that you have received permission from the bill payer for
+              using the app.
+            </p> 
+            <p>
+              Along the same lines, Open San Diego Org cannot always
```
Comment (NickEngmann) [inline p2plending/frontend/src/app/terms/Terms.js]: Replace "Open San Diego Org" with "Open San Diego"

File: p2plending/frontend/src/app/terms/Terms.js [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+              take responsibility for the way you use the app i.e. You
+              need to make sure that your device stays charged and if it
+              runs out of battery and you can't turn it on to avail the
+              Service, Open San Diego Org cannot accept
+              responsibility.
+            </p> 
+            <p>
+              With respect to Open San Diego Org's responsibility for
```
Comment (NickEngmann) [inline p2plending/frontend/src/app/terms/Terms.js]: Replace "Open San Diego Org" with "Open San Diego"

File: p2plending/frontend/src/app/terms/Terms.js [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+              from time to time. Thus, you are advised to review this page
+              periodically for any changes. We will
+              notify you of any changes by posting the new Terms and
+              Conditions on this page. These changes are effective
+              immediately after they are posted on this page.
+            </p> 
+            <p><strong>Contact Us</strong></p> <p>
+              If you have any questions or suggestions about
+              our Terms and Conditions, do not hesitate to
+              contact us at https://opensandiego.org/
```
Comment (NickEngmann) [inline p2plending/frontend/src/app/terms/Terms.js]: Hyperlink the "https://opensandiego.org" text to link to our website

File: p2plending/frontend/src/app/terms/Terms.js (hunk 2 above)
Comment (NickEngmann) [reply p2plending/frontend/src/app/terms/Terms.js]: Also lets hyperlink this first reference to Open San Diego to our website (https://opensandiego.org)

**Comment by CarlGraff** [comment]
Made changes you requested and added links to contact us on both the Terms page and the Privacy page:
image

image

File: p2plending/frontend/src/app/terms/Terms.js (hunk 3 above)
Comment (CarlGraff) [reply p2plending/frontend/src/app/terms/Terms.js]: done

File: p2plending/frontend/src/app/terms/Terms.js (hunk 4 above)
Comment (CarlGraff) [reply p2plending/frontend/src/app/terms/Terms.js]: done

File: p2plending/frontend/src/app/terms/Terms.js (hunk 5 above)
Comment (CarlGraff) [reply p2plending/frontend/src/app/terms/Terms.js]: done

File: p2plending/frontend/src/app/terms/Terms.js (hunk 6 above)
Comment (CarlGraff) [reply p2plending/frontend/src/app/terms/Terms.js]: done

File: p2plending/frontend/src/app/terms/Terms.js (hunk 1 above)
Comment (CarlGraff) [reply p2plending/frontend/src/app/terms/Terms.js]: done

File: p2plending/frontend/src/app/terms/Terms.js (hunk 2 above)
Comment (CarlGraff) [reply p2plending/frontend/src/app/terms/Terms.js]: done

**Pull request merged**